### DISCUSSION TITLE
Mehrere Buttons nebeneinander

### DISCUSSION THREAD

**Discussion by hopedotz** [Zefau/ioBroker.jarvis#870] [Ideas / Feature Requests]
Es wäre super, wenn man mehrere Buttons nebeneinander setzen könnte. Ich habe einige Licht- und Rolloszenen, die ich mittels eines Buttons ansteuere. Leider kann man die Buttons nur untereinander, aber nicht nebeneinander anordnen. Damit verschwendet man eigentlich unnötig Platz.

**Comment by Zefau** [comment]
https://github.com/Zefau/ioBroker.jarvis/issues/941

**Comment by mcuiobroker** [comment]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/system-effektprog./blindlevelupdown
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/system-effektprog./blindlevelbuttons
